release-notes/5.0/5.0-contributor-list.md

```diff
@@ -1,1176 +1,1176 @@
+</br>[Chris R (5)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=Tratcher@outlook.com)
+</br>[Chris Ross (4)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=chrross@microsoft.com)
+</br>[Chris Ross (69)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=Tratcher@Outlook.com)
+</br>[Chris Sainty (2)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=chrissainty@users.noreply.github.com)
+</br>[Chris Sienkiewicz (3)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=chsienki@microsoft.com)
+</br>[Christian Kadluba (1)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=10721825+ckadluba@users.noreply.github.com)
+</br>[Christian Wenz (1)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=christian@wenz.org)
+</br>[Christian Wenz (2)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=info@christianwenz.de)
+</br>[Christopher Haws (1)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=cribs2@gmail.com)
+</br>[Cédric Luthi (2)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=cedric.luthi@gmail.com)
+</br>[Dairai Nyabando (9)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=dnyabando@g.harvard.edu)
+</br>[Dan Friedman (1)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=dan@fantasticfriedmans.com)
+</br>[Dan Moseley (1)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=danmose@microsoft.com)
+</br>[Daniel Abbatt (1)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=daniel.abbatt@mail.com)
+</br>[Daniel Eisterhold (1)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=deisterhold@users.noreply.github.com)
+</br>[Daniel Roth (6)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=daroth@microsoft.com)
+</br>[David Fowler (29)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=davidfowl@gmail.com)
+</br>[David Melendez (1)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=dlmelendez@users.noreply.github.com)
+</br>[davidnx 44079650+davidnx@users.noreply.github.com (1)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=44079650+davidnx@users.noreply.github.com)
+</br>[Dawid Szmidka (1)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=dawid.szmidka@gmail.com)
+</br>[Dennis Daume (1)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=dennis@getmimo.com)
+</br>[DotNet Bot (21)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=dn-bot@microsoft.com)
+</br>[dotnet-maestro-bot dotnet-maestro-bot@microsoft.com (5)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=dotnet-maestro-bot@microsoft.com)
+</br>[dotnet-maestro[bot] 42748379+dotnet-maestro[bot]@users.noreply.github.com (755)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=42748379+dotnet-maestro[bot]@users.noreply.github.com)
```

Comment:
Thoughts on removing our well known bots like `dotnet-maestro`?

Reply:
bots from this list are removed